### PR TITLE
Fixes#284 Provide info about suitable size and resolution of background PNG

### LINKED ISSUE #284: Provide info about suitable size and resolution of background png
For the field "Upload a PNG file for badge background" it is unclear what size or ratio the png should have. Please provide info as a helper text below the field about best size, ratio and resolution.

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #284 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-  provided info as a helper text below the field about best size and resolution.

image

### CODE REVIEW

**Comment by ParthS007** [comment]
Ok, will do that :+1:

File: app/templates/index.html [hunk 1]
```diff
@@ -43,9 +43,11 @@
                         <label>Choose your Badge Background</label>
                         <ul style="list-style-type:none">
                             <li>
-                                <input type="radio" name="backsource" id="pngupload">Add PNG file</li>
-								<section id="upimage-input" style="display: none;">
-									<label for="imageFile">Upload a PNG file for badge background</label>
+                                <input type="radio" name="backsource" id="pngupload">Add PNG file
+                                <button type="button" style="background-color:#5BC0DE; font-style:italic; font-weight: bold; border-radius: 85%" data-toggle="tooltip" data-placement="right"
```
Comment (vaibhavsingh97) [inline app/templates/index.html]: Please refrain from inline styling. Please visit [here](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846) for more more info

File: app/templates/layout.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <title>BadgeYay</title>
     <link rel="stylesheet" href="{{ url_for('static', filename='css/bootstrap.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='css/jquery.minicolors.css') }}">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```
Comment (vaibhavsingh97) [inline app/templates/layout.html]: Please don't import bootstrap twice

Comment (vaibhavsingh97) [reply app/templates/layout.html]: Remove this

File: app/templates/layout.html [hunk 3]
```diff
@@ -17,6 +18,7 @@
     <script src="{{ url_for('static', filename='js/bootstrap.js') }}"></script>
     <script src="{{ url_for('static', filename='js/jquery.minicolors.min.js') }}"></script>
     <script src="{{ url_for('static', filename='js/script.js') }}"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
```
Comment (vaibhavsingh97) [inline app/templates/layout.html]: Same here

File: app/templates/index.html (hunk 1 above)
Comment (ParthS007) [reply app/templates/index.html]: I know this and I also don't want to use inline styling but when I am doing it by using a class, it is not getting priority and there is normal button shown, that's why I have used inline .

**Comment by ParthS007** [comment]
Thanks @djmgit  for suggestions , will do this. :+1:

**Pull request merged**